libs/plugins/scully-plugin-playwright/README.md

```diff
@@ -0,0 +1,7 @@
+# plugins-scully-plugin-playwright
+
```

Comment:
A bit of an update might be a wise idea, as we are going to automaticly pull those in?